add-isWebp-JS-style

Kretsul-Vitaliy opened the pull request:
Add JS class webp no-webp. Add retina background style mobile, and add style webp npwebp background.

Kretsul-Vitaliy committed:
add-isWebp-JS-style

Add JS class webp no-webp. Add retina background style mobile, and add style webp npwebp background.

Vlad-Moskalenko reviewed (approved):
відцентруй бекграунд

Kretsul-Vitaliy committed:
fix-position-background

Add new background desktop, delete old background dekstop, center position background desktop, tablet position fix 2 background.
